content/rs/references/metrics/auto-tiering.md

```diff
@@ -17,6 +17,9 @@ These metrics are additional metrics for [Auto Tiering ]({{< relref "/rs/databas
 #### % Values in RAM
 
 Percent of keys whose values are stored in RAM.
+Low precentage alert means most of the RAM is used for holding keys and not much RAM is avialble for values (working space).
+This can be due to high number of small keys, large keys. Insetering more keys might cause the DB to reach OOM error.
```

Comment:
```suggestion
This can be due to a high number of small keys or a few large keys. Inserting more keys might cause the database to run out of memory.
```